src/Tools/AI Test Toolkit/src/CommandLineCard.Page.al

```diff
@@ -0,0 +1,258 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+using System.Environment;
+using System.TestTools.TestRunner;
+using System.Utilities;
+
+page 149042 "CommandLine Card"
+{
+    Caption = 'AI Test CommandLine Runner';
+    PageType = Card;
+    Extensible = false;
+    ApplicationArea = All;
+
+    layout
+    {
+        area(content)
+        {
+            group(General)
+            {
+                field("AIT Suite Code"; AITCode)
+                {
+                    Caption = 'AIT Suite Code', Locked = true;
+                    ToolTip = 'Specifies the ID of the suite.';
+                    TableRelation = "AIT Test Suite".Code;
+
+                    trigger OnValidate()
+                    var
+                        AITTestSuite: record "AIT Test Suite";
+                    begin
+                        if not AITTestSuite.Get(AITCode) then
+                            Error(CannotFindAITSuiteErr, AITCode);
```

Comment:
use `this.` for AITCode
Consider making CannotFindAITSuiteErr local, otherwise also use `this.`